cif_core.dic

```diff
@@ -19286,6 +19286,593 @@ save_ATOM
+         loop_
+         _atom_analytical_extra.id
+         _atom_analytical_extra.meas_id
+         _atom_analytical_extra.mass_loss_percent
+         _atom_analytical_extra.analysis_temperature
+         LOD1 a   2  328
+         LOI1 a   5  623
+         LOI2 a  10 1023
+         LOI3 a  15 1373
+         LOI4 b   5  673
+         LOI5 b  10 1123
+
+;
+    _description_example.detail
+;
+         Four mass-loss percentages are given for measurement 'a', and two
+         for measurement 'b'. The mass losses were recorded after exposing
+         the specimen to the listed temperatures. The mass lost at a lower
+         temperature for the same measurement should be included in the
+         higher temperatures; that is, for measurement 'a', the total mass
+         loss after four measurements is 15 wt%, not (2+5+10+15) wt%.
+;
+
+save_
+
+save_atom_analytical_extra.analysis_temperature
+
+    _definition.id                '_atom_analytical_extra.analysis_temperature'
+    _definition.update            2022-11-21
+    _description.text
+;
+    The temperature, in kelvin, at which the mass loss was recorded
+    as given by _atom_analytical_extra.mass_loss_percent.
+
+    This would be used to record the temperature of drying or ignitition,
+    during, for example, fusion bead preparation for XRF analysis.
+;
+    _name.category_id             atom_analytical_extra
+    _name.object_id               analysis_temperature
+    _type.purpose                 Measurand
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            0.0:
+    _units.code                   kelvins
+
+save_
+
+save_atom_analytical_extra.analysis_temperature_su
+
+    _definition.id
+        '_atom_analytical_extra.analysis_temperature_su'
+    _definition.update            2022-11-21
+    _description.text
+;
+    Standard uncertainty of _atom_analytical_extra.analysis_temperature.
+;
+    _name.category_id             atom_analytical_extra
+    _name.object_id               analysis_temperature_su
+    _name.linked_item_id          '_atom_analytical_extra.analysis_temperature'
+    _units.code                   kelvins
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_atom_analytical_extra.id
+
+    _definition.id                '_atom_analytical_extra.id'
+    _definition.update            2022-11-21
+    _description.text
+
+;
+    Arbitrary label uniquely identifying the source of an elemental
+    composition value. This value is used by _atom_analytical.meas_id
+    to link individual composition values to their corresponding
+    technique of determination.
+;
+    _name.category_id             atom_analytical_extra
+    _name.object_id               id
+    _type.purpose                 Key
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
```

Comment:
Same issue as before -- `Code` is a good choice for the content type of an ID since it is case insensitive ("a" == "A'). Maybe `Word` would be better.

Reply:
done